### ISSUE TITLE
Rendering error (not a cyclic molecule)

### ISSUE THREAD

**Issue by Garuda1** [lambdacasserole/avogadrio#28]
Hey so as promised I kept bug testing and found a new rendering bug affecting non-cyclic molecules.

**IUPAC name:** tri(6-phenylphenyl)amine

**Render:** https://i.imgur.com/mRAlgtq.png

The problem here just seems to be that there isn't enough room for the external phenyl groups to display so they just overlap.

EDIT: technically, it contains cyclic groups, but the error doesn't seem to come from a massive cyclic chain unlike the previous ones.

**Comment by MFernflower** [comment]
Seems to be a common issue with chemistry software try typing that into http://opsin.ch.cam.ac.uk/ and it does the same thing

**Comment by lambdacasserole** [comment]
This is starting to become clearer now. The web application @MFernflower linked (http://opsin.ch.cam.ac.uk/) is also powered by the Indigo Cheminformatics Toolkit just like Sourire (the API wrapper behind Avogadrio) is. Issues of this kind related to rendering might be better off on the Indigo issues page (https://github.com/epam/Indigo/issues) if this isn't something we'd be able to fix by messing around with any Indigo settings (which I've tried to no avail). Closing for now.
